### ISSUE TITLE
Update to 1.15?

### ISSUE THREAD

**Issue by yurividal** [CampbellSoftwareSolutions/docker-osticket#87]
Any chance this image can be updated to 1.15 and/or have a build for 1.15 created?

**Comment by dm9bbadd4** [comment]
I second this request.

**Comment by bobweston** [comment]
Have you guys tried cloning this repo, bumping the version specified here

[https://github.com/CampbellSoftwareSolutions/docker-osticket/blob/master/Dockerfile#L3](https://github.com/CampbellSoftwareSolutions/docker-osticket/blob/master/Dockerfile#L3)

Then rebuilding the image?

`docker build --network host --no-cache -t new-os-ticket .`

Then just use that new-os-ticket image to start your container?

I don't work on this project. (and haven't gotten around to testing whether this works). But it is a pretty simple way to check.

FWIW, I have not seen a lot of responses to tickets in this repo.

**Comment by carcuevas** [comment]
Hi,

I can confirm that just changing this in the `Dockerfile` it will work:

```
ENV OSTICKET_VERSION=1.15.1
```

I built the image and test it and it is working ;-)

**Comment by akrea** [comment]
Good advice I tried that. Unfortunately, I cannot get it up and working. I always get
`/usr/local/bin/docker-php-entrypoint: exec: line 9: /data/bin/start.sh: not found`
the file shoud be at the right place - but I cannot verify it as I cannot get the container to work... If anybody could point me in the right direction would be helpful.

**Comment by akrea** [comment]
I found a solution. I posted it on [Docker Forums](https://forums.docker.com/t/container-wont-work-file-not-found/119903/3).

**Comment by xender69** [comment]
hi,
so if I change the ENV OSTICKET_VERSION=1.17.2 will that work as well?

**Comment by bobweston** [comment]
> hi,
> so if I change the ENV OSTICKET_VERSION=1.17.2 will that work as well?

I don't believe so. If you go to the OSTicket support pages, they're saying that version 1.17.x will be a complete rewrite of the code base. So I'd not assume that this will work out of the box.

Have you tried spinning up a docker image and trying it?

**Comment by MrDrache333** [comment]
See #93 for Support of Version 1.17.5
